src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/NotificationsService.cs

```diff
@@ -57,22 +55,32 @@ public async Task NotifySubscribersIfApplicable(params Guid[] releaseIds)
             }
         }
 
-        private IEnumerable<Publication> GetPublications(IEnumerable<Guid> releaseIds)
+        private async Task<PublicationNotificationMessage> BuildPublicationNotificationMessage(Release release)
         {
-            return _context.Releases
-                .AsNoTracking()
```

Comment:
You could add`.AsNoTracking` into the database access statement in `NotifySubscribersIfPossible`, e.g. `_context.Releases.AsNoTracking`...

Reply:
I've added `AsNoTracking` to the DB call in `NotifySubscribersIfApplicable`

Reply:
After doing this, it seems this prevents the `_context.Entry(release)[...].LoadAsync()`s from working, so I've removed `AsNoTracking` again.